modules/insticatorBidAdapter.js

```diff
@@ -64,17 +66,19 @@ function buildImpression(bidRequest) {
     });
   }
 
+  const gpid = deepAccess(bidRequest, 'ortb2Imp.ext.gpid');
```

Comment:
I noticed that master does not do this, but I found https://github.com/prebid/Prebid.js/pull/8189 which should bring them in sync. LGTM.

Reply:
8189 has been merged